test/minikube/minikube_base_database_test.go

```diff
@@ -419,56 +422,91 @@ func TestKubernetesRestoreDatabase(t *testing.T) {
 			"/opt/nuodb/bin/nuosql",
 			"--user", "dba",
 			"--password", "secret",
 			"demo",
-			"--file", "/opt/nuodb/samples/quickstart/sql/create-db.sql")
+			"--file", "/opt/nuodb/samples/quickstart/sql/create-db.sql",
+		)
+
+		// verify that the database contains the populated data
+		tables, err := testlib.RunSQL(t, namespaceName, admin0, "demo", "show schema User")
+		assert.NilError(t, err, "error running SQL: show schema User")
+		assert.Check(t, strings.Contains(tables, "HOCKEY"), "tables returned: ", tables)
 
 		// run a manual backup
-		k8s.RunKubectl(t, opts,
-			"exec", admin0, "--",
-			"nuodocker", "backup", "database",
-			"--db-name", "demo",
-			"--type", "full",
-			"--timeout", "120",
-			"--backup-root", "/var/opt/nuodb/backup",
-		)
+		// k8s.RunKubectl(t, opts,
+		// 	"exec", admin0, "--",
+		// 	"nuodocker", "backup", "database",
+		// 	"--db-name", "demo",
+		// 	"--type", "full",
+		// 	"--timeout", "120",
+		// 	"--backup-root", "/var/opt/nuodb/backup",
+		// )
+
+		// // trigger a manual backup now
+		// k8s.RunKubectl(t, opts,
+		// 	"exec", admin0, "--",
+		// 	"nuocmd", "set", "value", "--key", fmt.Sprintf("%s/demo/cluster0", testlib.BACKUP_SEMAPHORE), "--value", "true", "--unconditional",
+		// )
+
+		// wait for backup to complete
+		// testlib.Await(t, func() bool {
+		// 	output, err := k8s.RunKubectlAndGetOutputE(t, opts,
+		// 		"exec", admin0, "--",
+		// 		"nuocmd", "get", "value", "--key", fmt.Sprintf("%s/demo/cluster0", testlib.BACKUP_SEMAPHORE),
+		// 	)
+		// 	assert.NilError(t, err, "Error getting value from KV")
+		// 	return output == ""
+		// }, 240*time.Second)
+
+		// // wait for the backup to complete - up to 4 mins to start, and another 4 mins to then complete...
+		// databaseName := "demo"
+		// backupJob := fmt.Sprintf("hotcopy-%s-job-initial-", databaseName)
+		// testlib.AwaitPodPhase(t, namespaceName, backupJob, corev1.PodSucceeded, 120*time.Second)
 
 		// populate some more data
-		k8s.RunKubectl(t, opts,
-			"exec", admin0, "--",
-			"/opt/nuodb/bin/nuosql",
-			"--user", "dba",
-			"--password", "secret",
-			"demo",
-			"--file", "/opt/nuodb/samples/quickstart/sql/Teams.sql",
-		)
+		// k8s.RunKubectl(t, opts,
+		// 	"exec", admin0, "--",
+		// 	"/opt/nuodb/bin/nuosql",
+		// 	"--user", "dba",
+		// 	"--password", "secret",
+		// 	"demo",
+		// 	"--file", "/opt/nuodb/samples/quickstart/sql/Teams.sql",
+		// )
 
 		defer testlib.Teardown(testlib.TEARDOWN_RESTORE)
 		restoreDatabase(t, namespaceName)
 
+		// NTJ - Now automatically triggered by the restore chart (by default - can be overridden)
 		// and restart database - to trigger the restore
-		k8s.RunKubectl(t, opts,
-			"exec", admin0, "--",
-			"nuocmd", "shutdown", "database",
-			"--db-name", "demo",
-		)
-
+		// k8s.RunKubectl(t, opts,
+		// 	"exec", admin0, "--",
+		// 	"nuocmd", "shutdown", "database",
+		// 	"--db-name", "demo",
+		// )
+
+		// wait for the datbase to be restarted (shutdown followed by restart)
```

Comment:
typo in database

Reply:
No I am referring to datbase - as distinct to disbase... :)